samples/airflow/README.md

```diff
@@ -0,0 +1,44 @@
+
+### Warning 
```

Comment:
Let's make this README similar to the [Cloud Run](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run) or [Cloud Functions](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/functions) samples. We should add a title 'Data Validation on Airflow' and explain how to deploy this on Airflow/Composer. 

Under the high level overview we can include this 'Warning' section. That way the README is a more complete documentation for the Airflow sample code.